src/react-apps/applications/runtime/src/features/serviceInfo/containers/index.tsx

```diff
@@ -0,0 +1,125 @@
+import Grid from '@material-ui/core/Grid';
+import { createStyles, withStyles } from '@material-ui/core/styles';
+import * as React from 'react';
+import AltinnButton from '../../../../../shared/src/components/AltinnButton';
+import { get } from '../../../utils/networking';
+import Header from '../../header/containers';
+
+const styles = () => createStyles({
+  container: {
+
+  },
+});
+
+function ServiceInfo(props: any) {
+  const [reportee, setReportee] = React.useState(null);
+  const { history } = props;
+
+  const fetchReportee = async () => {
+    let routePrefix: string = null;
+    if (window.location.origin.includes('altinn.studio') || window.location.origin.includes('altinn3.no')) {
+      routePrefix = '/runtime';
+    }
+    const url: string = `${window.location.origin}${routePrefix}/api/v1/profile/user`;
+    const fetchedReportee: any = await get(url);
+    setReportee(fetchedReportee);
+  };
+
+  React.useEffect(() => {
+    if (!reportee) {
+      fetchReportee();
+    }
+  }, []);
+
+  const onClickInstantiate = () => {
+    history.push('/instantiate');
+  };
+
+  const { classes } = props;
+  return (
+    <>
+      <Header />
+      <Grid
+        container={true}
+        classes={classes}
+      >
+        {`Hei${reportee ? ' ' + reportee.party.person.firstName : null}!`}<br />
+        {`Du leverer skjema som `} <br />
```

Comment:
Hardcoded strings?

Reply:
Yeah, these values will be changed when we do a "implement instantiate design". 😅